src/Apps/Order/orderRoutes.tsx

```diff
@@ -127,7 +127,6 @@ export const orderRoutes: AppRouteConfig[] = [
             { order }
           )
 
-          // FIXME RELAY_UPGRADE
```

Comment:
I guess this still looks relevant: there's some code here that would need to be walked back.

Reply:
Wasn't able to solve this one, but this maintains that the logic we were testing as valid within the test, albeit annoyingly